PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -326,6 +347,29 @@ then
     log_Err_Abort "--printcom is not consistently implemented, do not rely on it"
 fi
 
+# Convert the --runmode string argument into a numeric code
+case "$RunMode" in
+    Default|"")
+        RunMode=1
+        ;;
+    ACPCAlignment)
+        RunMode=2
+        ;;
+    BrainExtraction)
+        RunMode=3
+        ;;
+    T2wToT1wRegAndBiasCorrection)
+        RunMode=4
+        ;;
+    AtlasRegistration)
+        RunMode=5
+        ;;
+    *)
+        echo "Error: invalid runmode '$RunMode'"
+        exit 1
```

Comment:
```suggestion
        log_Err_Abort "Error: invalid runmode '$RunMode'"
```

Also, below this block, add either an `if [[ "$BrainExtract" != INVIVO && "$BrainExtract" != EXVIVO ]]` or a `case "$BrainExtract" in` to make the script error if the user has done something silly like `--brainextract=UNKNOWN`.